vector/src/main/java/im/vector/app/features/widgets/WidgetFragment.kt

```diff
@@ -75,7 +82,7 @@ class WidgetFragment @Inject constructor() :
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         setHasOptionsMenu(true)
-        views.widgetWebView.setupForWidget(this)
+        views.widgetWebView.setupForWidget(this, this)
```

Comment:
We should avoid having 2 parameters referencing as `this`

Reply:
Good call. I've introduced a compound interface to avoid this now.